CoverMyDotNet.Samples/Program.cs

```diff
@@ -57,16 +57,48 @@ public static void MultipleRequests(Client client)
 			var requests = client.GetRequests(tokens.ToArray());
 			foreach(var v in requests.Requests)
 			{
-				Console.WriteLine("Patient: {0} {1}", v.Patient.FirstName, v.Patient.LastName);
+				Console.WriteLine("Token: {0} {1}", v.Id, v.Tokens.First().Id);
 			}
 		}
+
+		public static void DrugSearch(Client client)
```

Comment:
What are your thoughts on combining the drug-related examples into one class, and ditto for form ones?  Then perhaps extracting each into their own files so that Program.cs just has the Main class in it??  I haven't done any .NET programming in awhile, but IIRC correctly that's a common practice.